assets/src/design-system/components/pill/index.js

```diff
@@ -52,6 +52,7 @@ const StyledPill = styled.button(
 
     &:disabled {
       pointer-events: none;
+      color: ${isActive ? theme.colors.bg.primary : theme.colors.fg.disable};
```

Comment:
We could use the `:active` pseudo selector instead of the `isActive` prop if we'd like here 🤷. Easier since we leverage the browser functionality

I may be misunderstanding, but `isActive` may refer to a specific pill state rather than the button itself being active. In that case, I think we should rename this to something else 🤔. Maybe `isEnabled` or `isSelected`

Reply:
🔑  renaming this is a good move, it's not actually when the pill itself is `:active` - you're right. that's confusing.

Reply:
I'm actually going to leave this here because it is the _active_ state of the UI, the pill that is selected.